torch/distributions/binomial.py

```diff
@@ -25,26 +21,30 @@ class Binomial(Distribution):
          100
         [torch.FloatTensor of size 4]]
 
+        >>> m = Binomial(torch.Tensor([[5.], [10.]]), torch.Tensor([0.5, 0.8]))
+        >>> x = m.sample()
+         4  5
+         7  6
+        [torch.FloatTensor of size (2,2)]
+
     Args:
-        total_count (int): number of Bernoulli trials
+        total_count (Tensor): number of Bernoulli trials
```

Comment:
This has to be a tensor because the counts might vary.

Reply:
I meant that you should treat a Python scalar as a constant tensor. That's what it used to mean, and it should remain like that, because it's generally useful

Reply:
Yes, we can pass in either scalars or tensor so this shouldn’t break backwards compatibility.

Reply:
Will update the docs accordingly.